src/Bielu.Examine.Core/Indexers/ElasticSearchBaseIndex.cs

```diff
@@ -23,7 +23,7 @@ public class ElasticSearchBaseIndex(
     private ExamineIndexState IndexState => indexStateService.GetIndexState(name);
     private static readonly object _existsLocker = new object();
 
-    private IDisposable Unsubscriber => elasticSearchService.Subscribe(this);
```

Comment:
afaik it is required in Observer patter, but I am not 100% sure about it.

Reply:
The problem is that is wat complaining that Unsubscriber was readonly. I could not compile it this way. In the Microsoft example they declare it like this as well: https://learn.microsoft.com/en-us/dotnet/standard/events/how-to-implement-an-observer#example